plugins/explore/src/components/ExplorePluginPage.tsx

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { FC } from 'react';
+import { makeStyles, Typography } from '@material-ui/core';
+import {
+  Content,
+  ContentHeader,
+  Header,
+  Page,
+  pageTheme,
+  SupportButton,
+} from '@backstage/core';
+import ExploreCard, { CardData } from './ExploreCard';
+import { BackstageTheme } from '@backstage/theme';
+
+const useStyles = makeStyles<BackstageTheme>(theme => ({
+  container: {
+    display: 'grid',
+    gridTemplateColumns: 'repeat(auto-fill, 296px)',
+    gridGap: theme.spacing(3),
+    marginBottom: theme.spacing(6),
+  },
+}));
+
+const toolsCards = [
```

Comment:
Ideally this data should be fetched directly from the plugins included in the Backstage deployment.

Reply:
With GitHub auth we could fetch info about existing plugins from GH, and maybe also include `[Plugin]` issues under an ideas/WIP section?

Reply:
Depends on what the purpose of this plugin is i guess. Who will consume the views? I thought this was for end users to easily find functionality and features.

Reply:
We should def get backstage instance for managing this project up at some point, and this could be a part of that.

Reply:
So meta.

Yes that instance could even have a marketplace of sorts, but it would arguably be a different plugin than this one.

Reply:
#697 

I don't see why we wouldn't want to promote WIP plugins at this point though, as in just include those in the exploration too.

Reply:
I'm probably misunderstanding you, sorry for any confusion. I was just thinking that wip plugin promotion may not be helpful for potentially hundreds of thousands (dream big!) of end user Backstage employees out in the world, that have nothing to do with the actual implementation of the system internally.

Reply:
I see different uses of this depending on: 

Production - only plugins in your deployment of Backstage shows up. This feature helps users of Backstage find tools available. 

Dev - All open sourced plugins appear, helping Operators see what plugins are available. 

I think we should have a demo Dev instance at some point. Ping @decause